doc/usage/extensions/autodoc.rst

```diff
@@ -662,6 +662,13 @@ There are also config values that you can set:
    .. __: https://mypy.readthedocs.io/en/latest/kinds_of_types.html#type-aliases
    .. versionadded:: 3.3
 
+.. confval:: autodoc_unqualified_typehints
```

Comment:
I'm still not sure that a boolean value is the best design for such a purpose. It might be better to use enum style:
```
autodoc_typehints_style = "fully-qualified" | "unqualified" | ...
```

Reply:
I think it's not late to change this configuration to enum style after we'll find another setting of typehints format. So I'm going to go with this.